tokens/src/lib.rs

```diff
@@ -2135,7 +2135,6 @@ where
 }
 
 impl<T: Config> TransferAll<T::AccountId> for Pallet<T> {
-	#[transactional]
```

Comment:
I think we can keep `transactional` for trait implementations, which could be used in `on_initialize/finalize` which are not `transactional` on default.